### PR TITLE
Fix height and width sigma and kernel axis order in gaussian_blur

### LINKED ISSUE #127247: tf.keras.ops.image.gaussian_blur appears to swap height and width sigma parameters
### Issue type

Bug

### Have you reproduced the bug with TensorFlow Nightly?

Yes

### Source

source

### TensorFlow version

tf 2.21.0, also reproduced on tf 2.22.0-dev20260904

### Custom code

Yes

### OS platform and distribution

Linux Ubuntu 22.04

### Mobile device

_No response_

### Python version

Python 3.13.5

### Bazel version

_No response_

### GCC/compiler version

_No response_

### CUDA/cuDNN version

_No response_

### GPU model and memory

_No response_

### Current behavior?

`tf.keras.ops.image.gaussian_blur` returns an incorrect forward value when distinct height and width sigma values are provided.

The API documents `sigma` in height,width order. In the reproducer below, `sigma=(0.5, 2.0)` is used with a nonsymmetric `3 x 3` image. Only the center pixel is inspected, so the result does not depend on padding behavior.

The expected center value using height sigma `0.5` and width sigma `2.0` is approximately `-1106.2184462981334`. However, TensorFlow returns `-1242.3213516003145`, which is consistent with applying the sigma values to the opposite axes.

Expected behavior?

`tf.keras.ops.image.gaussian_blur` should apply `sigma=(0.5, 2.0)` in height,width order. For the reproducer below, the center output should be close to `-1106.2184462981334`, not `-1242.3213516003145`.

### Standalone code to reproduce the issue

```shell
import os
os.environ["CUDA_VISIBLE_DEVICES"] = ""
os.environ["OMP_NUM_THREADS"] = "1"
os.environ["TF_NUM_INTRAOP_THREADS"] = "1"
os.environ["TF_NUM_INTEROP_THREADS"] = "1"

import tensorflow as tf

def target(t):
    image = tf.reshape(
        tf.stack([(i + 1) ** 2 * t - i for i in range(9)]),
        [3, 3, 1],
    )

    y = tf.keras.ops.image.gaussian_blur(
        image,
        kernel_size=(3, 3),
        sigma=(0.5, 2.0),
        data_format="channels_last",
    )

    return y[1, 1, 0]

x = tf.constant(-40.0, dtype=tf.float64)

actual = target(x)
expected = -1106.2184462981334

print("actual:", actual.numpy())
print("expected:", expected)

if abs(float(actual) - expected) > 1e-9:
    print("BUG REPRODUCED: tf.keras.ops.image.gaussian_blur appears to swap height and width sigma parameters")
else:
    print("not reproduced")
```

### Relevant log output

```shell
actual: -1242.3213516003145
expected: -1106.2184462981334
BUG REPRODUCED: tf.keras.ops.image.gaussian_blur appears to swap height and width sigma parameters
```

### PR DESCRIPTION
**Pull request by jdymitarai**
## Description

In `gaussian_blur`, `kernel_size` and `sigma` are documented and provided in `(height, width)` order:
```python
kernel_size: A tuple of two integers, specifying the height and width of the Gaussian kernel.
sigma: A tuple of two floats, specifying the standard deviation of the Gaussian kernel along height and width.
```

However, `_get_gaussian_kernel2d` across all backends previously treated index 0 as X (width) and index 1 as Y (height), and combined them via `outer(kernel1d_y, kernel1d_x)` (or `multiply(kernel1d_y, kernel1d_x)`).

Because dimension 0 of the resulting 2D convolution kernel corresponds to height and dimension 1 corresponds to width:
- Height (dim 0) received `size[1], sigma[1]` (the width parameters).
- Width (dim 1) received `size[0], sigma[0]` (the height parameters).

This caused height and width sigma values to be inverted on all backends (TensorFlow, PyTorch, JAX, NumPy, OpenVINO) as reported in tensorflow/tensorflow#127247, and caused non-square kernel shapes to be incorrectly transposed.

### Fix
1. Correctly maps `size[0], sigma[0]` to `kernel1d_h` and `size[1], sigma[1]` to `kernel1d_w`.
2. Constructs the 2D kernel with proper `(height, width)` dimensions via `outer(kernel1d_h, kernel1d_w)`.
3. Applied consistently across all backends:
   - `keras/src/backend/tensorflow/image.py`
   - `keras/src/backend/torch/image.py`
   - `keras/src/backend/jax/image.py`
   - `keras/src/backend/numpy/image.py`
   - `keras/src/backend/openvino/image.py`
4. Added `test_gaussian_blur_asymmetric_sigma` in `keras/src/ops/image_test.py` to assert correct directional filtering and non-square kernel handling.

Fixes tensorflow/tensorflow#127247

## Contributor Agreement

- [x] This PR is linked to an issue that has been assigned to me (see `Fixes #xxx` above).
- [x] I am a human, and not a bot.
- [x] I will be responsible for responding to review comments in a timely manner.
- [x] I will work with the maintainers to push this PR forward until submission.

### CODE REVIEW

**Comment by jdymitarai** [comment]
Closing this PR for now to investigate torch backend asymmetric kernel handling further off-line. Thanks!
